### PR TITLE
Added sec() function

### PR DESCRIPTION
**Pull request by titoausten**
- `sec()` is a trigonometry function that returns the reciprocal (not inverse) of `cos()` i.e `sec() = 1 / cos()`.

- Added new regression tests for `sec()`.

### CODE REVIEW

**Comment by jrgemignani** [comment]
@titoausten Can you justify needing this?

**Comment by titoausten** [comment]
It's a mathematical trigonometric function and I believe it's better to have a stand alone function than finding reciprocal of the `age_cos` function. AGE can be used for mathematical graph queries too and having such functions would make AGE's use easy.

**Comment by jrgemignani** [comment]
@titoausten Does Neo4j or PostgreSQL support what you want to add?

**Comment by titoausten** [comment]
> @titoausten Does Neo4j or PostgreSQL support what you want to add?

Yes, the `sec()` function is dependent on PostgreSQL's `dcos()` function in `float.c` file.

**Comment by titoausten** [comment]
> > > @titoausten Does Neo4j or PostgreSQL support what you want to add?
> > 
> > Yes, the `sec()` function is dependent on PostgreSQL's `dcos()` function in `float.c` file.
> 
> Let me rephrase that,... does Neo4j or PostgreSQL have a secant function?

Oh, none of them have a secant function and that's why I worked on it.
